### PR TITLE
Set up a "Hello world" project with rails-react gem

### PR DESCRIPTION
**Pull request by mahbubul14**
## Project requirements
* Create a new Rails app called 'hello-rails-react'.
* Initialize your project with Git.
* Set up your project with webpacker the react-rails gems as describe in the previous lesson's tutorial.
* Make sure that your project has a Postgres database set up.
* Create a table for storing your messages and create 5 different greetings.
* Create an API endpoint that selects a random greeting from your table (you will need a controller with an action and Rails route).
* Create a static view that will be the root of your app.
* Create the App component with react-router and render it in your static view.
* Create the Greeting component that will display a greeting. Set it up as a route in your App component.
* Create a store, an action and a reducer that will connect to you API endpont to get the random greeting.
* Display the random greeting in your Greeting component.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,24 @@
...
+* Database creation
+
+* Database initialization
+
+* How to run the test suite
+
+* Services (job queues, cache servers, search engines, etc.)
+
+* Deployment instructions
+
```
Comment (Deepakdanger) [inline README.md]: - [ ] I would recommend making the README professional with all the necessary details like title, description, steps to clone and run the project locally with proper step by step command, author details, etc.. You can follow the [template](https://github.com/microverseinc/readme-template) given by the microverse as well.

File: app/controllers/api/v1/greetings_controller.rb [hunk 2]
```diff
@@ -0,0 +1,8 @@
+class Api::V1::GreetingsController < ApplicationController
+  def index
+    @greetings = Greeting.all
+    # render json: @greetings, status: :ok
```
Comment (Deepakdanger) [inline app/controllers/api/v1/greetings_controller.rb]: - [ ] I would recommend removing the commented code to make the codebase cleaner.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @mahbubul14, 

Good job so far! 👏
There are some issues that you still need to work on to go to the next project but you are almost there!

## To Highlight :100: 
- No linter error :heavy_check_mark: 
- The application is working :+1: 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by rustam-tolipov — APPROVED** [review]
## Hi @mahbubul14 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 🎉 🎉 🎉 🎉 

Cheers and Happy coding!👏👏👏

image

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**